draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -344,7 +329,7 @@ This draft presents the modeling of the coherent pluggable such as (D) and (C) i
 
 The functional building blocks of the coherent pluggables of {{figure-details-packet-optical-device}} are shown in {{figure-optical-pluggable-building-blocks}} and has three major functions:
 
-- Media side: This functional block represents all Photonic/Optical attributes of the coherent pluggables (interface (5) in {{figure-details-packet-optical-device}}). These attributes define the characteristics of the optical and photonic properties such as spectrum, polarization, dispersion etc., which do not directly affect the behavior of the host packet device. Note that the goal of this draft is to eventually provide the YANG data model with these specific paramters which can be exposed in the model towards the controllers.
+- Media side: This functional block represents all Photonic/Optical attributes of the coherent pluggables (interface (5) in {{figure-details-packet-optical-device}}). These attributes define the characteristics of the optical and photonic properties such as spectrum, polarization, dispersion etc., which do not directly affect the behavior of the host packet device. Note that the goal of this draft is to eventually provide the YANG data model with these specific parameters which can be exposed in the model towards the controllers.
 
 - Host side: This functional block represents all Host/Electrical attributes of the coherent pluggables (interface (4) in {{figure-details-packet-optical-device}}). These attributes defines the characteristics of interconnect between the host and the optical pluggable, such as lane count, FEC etc., which both the optical pluggable and the packet host should understand and act upon.
 
```

Comment:
as there is no guaranteed 1:1 mapping between Media Logical Channels and Host Logical Channels, the figure should contain a mapping-block function in between.
Depending on the L0/L1 modeling discussion, we may need to structure this accordingly (FFS)